docs/source/newsfragments/4656.change.rst

```diff
@@ -0,0 +1 @@
+If the user requests coverage collection via :envvar:`COCOTB_LIBRARY_COVERAGE` or :envvar:`COCOTB_USER_COVERAGE` and the :mod:`coverage` module is not available, the regression will fail instead of continue.
```

Comment:
```suggestion
If the user requests coverage collection via :envvar:`COCOTB_LIBRARY_COVERAGE` or :envvar:`COCOTB_USER_COVERAGE` and the :mod:`coverage` module is not available, the regression will fail instead of continuing.
```

?

(or just remove the "instead of" part)

Reply:
```suggestion
If the user requests coverage collection via :envvar:`COCOTB_LIBRARY_COVERAGE` or :envvar:`COCOTB_USER_COVERAGE` and the :mod:`coverage` module is not available, the regression will fail.
```